java/test/TestH5D.java

```diff
@@ -966,25 +966,121 @@ public void testH5Dvlen_get_buf_size()
     }
 
     @Ignore
```

Comment:
Do you want to test these things in this test file or are they still to be ignored for some reason?

Reply:
I have left that one there as I am undecided on its usefulness. Maybe I should just merge the len check into the test that actually reads. I'm thinking the len check might not actually even need the H5write call.

Reply:
Missed this in the last update - will address in next PR.